File: ControlFlow/isPalindromeInteger.py
import math
def isPalindromeInteger(n):
    string = str(n)
    length = len(string)
    mid = math.floor(n / 2)

    for i in range(0, mid+1):
        if string[i] != string[length - 1 - i]:return False
        
    return True

File: ControlFlow/isPalindromeInteger.java
class Solution {
  public static boolean isPalindromeInteger(int n) {
    String s = String.valueOf(n);
    int l = s.length();
    int midle = (int) Math.floor(l / 2);

    for (int i = 0; i < midle; i++) {
      if (s.charAt(i) != s.charAt(l - 1 - i)) {
        return false;
      }
    }

    return true;
  }
}
